packages/trace-viewer/vite.sw.config.ts

```diff
@@ -35,9 +35,10 @@ export default defineConfig({
       '@web': path.resolve(__dirname, '../web/src'),
     },
   },
+  publicDir: false,
   build: {
-    outDir: path.resolve(__dirname, '../playwright-core/lib/vite/traceViewer'),
-    // Output dir is shared with vite.config.ts, clearing it here is racy.
+    // outputs into the public dir, where the build of vite.config.ts will pick it up
+    outDir: path.resolve(__dirname, 'public'),
     emptyOutDir: false,
```

Comment:
We can't remove this one, because we don't want it to delete the `public` directory. But i've removed the other one, there's no race anymore. https://github.com/microsoft/playwright/pull/33291/commits/29d21fda8d5bd770c172bd1581e0adb166e9f9b8